doc.go

```diff
@@ -382,6 +388,25 @@ of pigeon and should not be used nor modified. Those keys are treated as
 internal implementation details and therefore there are no guarantees given in
 regards of API stability.
 
+Left recursion
+
+With options -support-left-recursion pigeon supports left recursion. E.g.:
+	expr = expr '*' term
+Supports indirect recursion:
+	A = B / D
+	B = A / C
+Preserve support for ordered choices:
+	expr = expr '*' term / expr '+' term
+The implementation is based on the [Left-recursive PEG Grammars][9] article that
+links to [Left Recursion in Parsing Expression Grammars][10] and
+[Packrat Parsers Can Support Left Recursion][11] papers.
+
+References:
+
+    [9]: https://medium.com/@gvanrossum_83706/left-recursive-peg-grammars-65dab3c580e1
+	[10]: https://arxiv.org/pdf/1207.0443.pdf
+	[11]: http://web.cs.ucla.edu/~todd/research/pepm08.pdf
```

Comment:
consistent indentation with `tab`:

```suggestion
References:

	[9]: https://medium.com/@gvanrossum_83706/left-recursive-peg-grammars-65dab3c580e1
	[10]: https://arxiv.org/pdf/1207.0443.pdf
	[11]: http://web.cs.ucla.edu/~todd/research/pepm08.pdf
```

Reply:
Fixed